src/GeneralStateTestsFiller/stCodeSizeLimit/createInitCodeSizeLimitFiller.yml

```diff
@@ -0,0 +1,120 @@
+# CREATE with max allowed init code size
+createInitCodeSizeLimit:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '20000000'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 200000000
+      code: ''
+      nonce: 0
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: |
+        :yul 
+        {
+          // :yul { return(0, 1) }
+          mstore(0, 0x60016000f3000000000000000000000000000000000000000000000000000000) 
+          // get initcode size from calldata
+          sstore(0, create(0, 0, calldataload(0)))
+          sstore(1, 1)
+        }
+      nonce: 0
+      storage: {}
+
+  transaction:
+    data:
+      - ':label valid :raw 0x000000000000000000000000000000000000000000000000000000000000c000'
+      - ':label invalid :raw 0x000000000000000000000000000000000000000000000000000000000000c001'
+    gasLimit:
+      - 15000000
+    gasPrice: 10
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: 'b94f5374fce5edbc8e2a8697c15331677e6ebf0b'
+    value:
+    - 0
+
+  expect:
+    - indexes:
+       data: ':label valid'
+       gas: !!int -1
+       value: !!int -1
+      network:
+        - 'London'
+      result:
+       a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          nonce: 1
+          balance: 198932050
+       b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          storage:
+              '0': 'f1ecf98489fa9ed60a664fc4998db699cfa39d40'
+              '1': 1
+       f1ecf98489fa9ed60a664fc4998db699cfa39d40:
+          balance: 0
+          code: '0x00'
```

Comment:
empty code both for valid and invalid creations ?

Reply:
(it's not empty, but one byte `00` of code deployed) and yes before EIP result is the same 

With EIP account not created for invalid initcode.

Reply:
> (it's not empty, but one byte `00` of code deployed) and yes before EIP result is the same for valid and invalid.
> 
> With EIP account is not created for invalid initcode.

Reply:
I made it deploy the codecopied init code itself (`0x600a80600080396000f3`)